### DISCUSSION TITLE
Integrated Decision Unit (IDU)

### DISCUSSION THREAD

**Discussion by Achidago** [SERVIR-WA/GALUP#42] [Q&A] [answered]
What is the purpose of the Integrated Development Unit (IDU) ?

How did the GALUP team come by the various divisions within THLD?

**Comment by chjch** [comment, marked as answer]
Excellent question.

The **Integrated Decision Units** (IDUs) are a set of spatial containers, modeled as a set of polygon-based _Geographic Information System_ (GIS) coverages containing spatially explicit depictions of landscape attributes and patterns.

IDUs are created based on information from multiple sources, such as parcel-level information, floodplain delineations, and soil permeability. By using vector-based GIS including intersection, expansion, grouping, etc, we can merge the information together to generate a layer of IDUs. 

Since IDUs are created based on traits of landscape, the **area within an IDU is deemed as homogeneous**, so **_a single uniform decision can be made_** to the IDU, without worrying about conflicts caused by heterogeneity. 

IDU roots from agent-based modeling (ABM). It is a useful mechanism to determine **a spatially explicit delineation of a landscape** and making a land-use plan on top of them. This paper (linked below) is a specific use case of IDUs. [Modeling biocomplexity](https://www.sciencedirect.com/science/article/pii/S1364815206000296?via%3Dihub).

**Reply by Achidago** [reply]
Thanks for your  response. Well understood

**Reply by chjch** [reply]
Thanks. Please "Mark as answer" in the upper right corner of my response.

**Reply by mmunzamil** [reply]
Hello @chjch 

Can you please illustrate the step by step approach in creating the IDU for THL District in QGIS?

**Reply by chjch** [reply]
Hi Mohammed,

Thanks for expressing interest in the method of creating IDUs. <ins>Can you tell me more about why you are interested in know how exactly the IDUs were created</ins>? I want to learn more about the incentive so I can understand your needs. 

- Do you think this IDU concept is applicable to your routine work?
- Do you think it is useful for LUSPA, in general, to learn this technique? Why? @mmunzamil @Achidago (maybe both of you can answer these questions)

**P.S.**: (background) These IDUs were provided to have a set of polygons to work with for Modules and Exercises in this workshop and to illustrate the usage of _LUCIS-OPEN Tools for QGIS_. It was originally created inside ArcGIS, not QGIS. But, it is definitely replicable in QGIS.

**Reply by Achidago** [reply]
@chjch Personally, i will be interested in knowing how to develop IDUs based on some predetermined standards and  criteria to assess:
1.  Accessibiliyy of cities/towns/communities/neighbourhoods to services and  infrastructure
2. The impact of societal interventions on settlements/sections of settlements. 
3. Site suitabilty and produce applications/maps as decision support tools/systems e.g Crop farming, locating services/infrastructure etc

**Reply by mmunzamil** [reply]
Dear @chjch 
We used the fishnet tool (akin to the IDU concept) to estimate the rural population of communities in Ghana during the preparation of the National Spatial Development Framework. Some instances of analysis are shown in the diagrams below. This was however done in ArcGIS, and if it would take too long in QGIS, you might teach us how to do it in ArcGIS.
![image](https://user-images.githubusercontent.com/85182668/122698624-2f16f500-d237-11eb-823a-519f96420429.png)
![image](https://user-images.githubusercontent.com/85182668/122698707-5e2d6680-d237-11eb-8caf-42c704fd044b.png)
kind regards

**Comment by chjch** [comment]
Hi @mmunzamil 
Since the previous thread gets very long, I'd like to start a new thread about this topic here.
<ins>Thank you so much</ins> for including the two maps above. It helps us better understand your works at LUSPA.
I will definitely create a separate tutorial on how to create IDUs (@Achidago) with a concrete example at some point during this training. I think it could be a topic and a stand-alone training session when we visit LUSPA in person (perhaps in September). Along that line of thought, I believe this idea of IDUs should be emphasized and well discussed. To do that, I want to first begin by answering **why IDUs are needed**. 
### The meaning of a set of spatially explicit polygons to land use planning
IDUs essentially are a set of polygons with definite boundaries. We assign a specific use for each IDU (based on suitability analysis). This assignment is simply referred to as **_a land-use decision_**. The core concept of IDU is that land is deemed **homogeneous** within the area of a single IDU. Such homogeneity ensures a **_land-use decision_** is "good" for every piece of land within the IDU, rather than only a portion of the IDU. 
Since land-use decisions are intrinsically discrete, i.e., two adjacent parcels can have different uses, a vector (polygon) representation is more applicable than a raster representation. That's why we need to have a system of polygons, such as IDUs, to begin with. This is the first, however, critical step to achieving a spatially explicit land-use plan.
### What are the rules of creating IDUs
1. You don't need to always create new IDUs. They can be an existing layer of polygons that you are willing to use for making land-use decisions. For example, they can be a polygon layer that reflects property ownership.
2. IDUs tend to be smaller when in an urban context comparing to a rural or natural context because land-use patterns are more granular and sporadic in urban areas, which is needed to support various types of human activities (e.g., school, grocery, hospital).
3. Natural and artificial separations should be included in the creation of IDUs. That means IDUs should reflect and not in conflict with roads, streams, rivers, forests.

QGIS does have a similar function called [Create grid](https://docs.qgis.org/3.10/en/docs/user_manual/processing_algs/qgis/vectorcreation.html#create-grid) for "fishnet" creation. I also intend to develop a new function called "**Gridify**" and incorporate it as a new tool in **_LUCIS-OPEN Tools for QGIS_**. 
Although fishnet is useful and commonly used in the planning process, it bears shortcomings comparing to IDU. Since polygons in a fishnet are uniform, they cannot be adjusted to incorporate multiple sources of information, including elevation, soil condition, roads, etc. 

I will talk to my teammates and see the possibility of creating a dedicated session for a demo of creating IDUs during our in-person visit. I'll follow up on this.

**Reply by Achidago** [reply]
Thanks for the response @chjch. We look forward to it
